src/test/java/teammates/ui/webapi/CreateAccountActionTest.java

```diff
@@ -39,27 +39,20 @@ protected void testExecute() {
         String institute = "TEAMMATES Test Institute 1";
 
         ______TS("Not enough parameters");
-        AccountCreateRequest badRequest = buildCreateRequest(null, institute, email);
-
-        try {
-            getAction(badRequest).execute();
-        } catch (InvalidHttpRequestBodyException e) {
-            assertEquals("name cannot be null", e.getMessage());
-        }
-
-        badRequest = buildCreateRequest(name, null, email);
-        try {
-            getAction(badRequest).execute();
-        } catch (InvalidHttpRequestBodyException e) {
-            assertEquals("institute cannot be null", e.getMessage());
-        }
-
-        badRequest = buildCreateRequest(name, institute, null);
-        try {
-            getAction(badRequest).execute();
-        } catch (InvalidHttpRequestBodyException e) {
-            assertEquals("email cannot be null", e.getMessage());
-        }
+
+        Exception exception;
+
+        exception = assertThrows(InvalidHttpRequestBodyException.class,
```

Comment:
- This can combine with two lines before this
- Again this is nitpicking, but the variable name need not be this long; `e` or `ex` are generally understood as exception variable. More descriptive names may be necessary if there are multiple of such variables around, but right now there is just one.